src/bluecoins_cli/tui.py

```diff
@@ -30,35 +32,92 @@
             corner: '96'
 """
 
-with YamlLoader() as loader:
-    loader.load(PYTERMGUI_CONFIG)
-
-with WindowManager() as manager:
-    window = (
-        Window(
-            "",
-            InputField(
-                "A CLI tool to manage the database of Bluecoins,\nan awesome budget planner for Android.",
-                multiline=True,
-            ),
-            "",
-            Container(
-                "In development:",
-                InputField("- archive"),
-                box="EMPTY_VERTICAL",
-            ),
-            "",
-            [
-                "Convert",
-                lambda *_: call(
-                    ["python", ADB_SYNC_SCRIPT_PATH],
-                ),
-            ],
-            width=60,
-            box="DOUBLE",
+sync = SyncManager()
+DB = sync.prepare_local_db()
+
+
+def run_tui() -> None:
+
+    with YamlLoader() as loader:
+        loader.load(PYTERMGUI_CONFIG)
+
+    with WindowManager() as manager:
+        main_window = Window(width=60, box="DOUBLE")
+
+        window = (
+            (
+                main_window
+                + ""
+                + Label(
+                    "A CLI tool to manage the database of Bluecoins,\nan awesome budget planner for Android.",
+                )
+                + ""
+                + Container(
+                    "In development:",
+                    Label("- archive"),
+                    box="EMPTY_VERTICAL",
+                )
+                + ""
+                + Button('Convert', lambda *_: manager.add(get_choose_currency_window(manager)))
+                + ""
+                + Button('Archive', lambda *_: manager.add(get_choose_account_archive_window(manager)))
+                + ""
+                + Button('Exit programm', lambda *_: close_session())
+                + ""
+            )
+            .set_title("[210 bold]Bluecoins CLI")
+            .center()
         )
-        .set_title("[210 bold]Bluecoins CLI")
-        .center()
-    )
 
-    manager.add(window)
+        manager.add(window)
+
+
+def get_choose_currency_window(manager: WindowManager) -> Window:
+    # FIXME: hardcode
+    base_currency = 'USD'
+    window = Window()
+
+    currency_window = (
+        window
+        + ""
+        + Button(base_currency, lambda *_: start_convert(base_currency))
+        + ""
+        + Button('Back', lambda *_: manager.remove(window))
+    ).center()
+
+    return currency_window
+
+
+def get_choose_account_archive_window(manager: WindowManager) -> Window:
+    # FIXME: hardcode
+    path_test = '/home/larnaa/VScode_project/bluecoins-cli/Bluecoins_last_3.fydb'
```

Comment:
I don't think we should merge it. Hardcode is okay for now, but not machine-specific.